#Goal
roomba_robot/PositionAndDestination destin
---
#Result
float32 time_spent
---
#Feedback
float32 distance
